packages/typescript/README.md

```diff
@@ -39,6 +39,22 @@ module.exports = {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
     },
+    // An alternative `tsconfig` file for tests is specified here so that test files can be linted
+    // without building them during the normal build process. This file should have `noEmit` set to
+    // `true`.
+    {
+      files: ['*.test.ts'],
+      parserOptions: {
+        project: './tsconfig.test.json',
```

Comment:
Not sure we need this. Our conventions have been drifting to using `tsconfig.json` as a `noEmit` config just for testing, and using `tsconfig.build.json` for building. In which case this section is not necessary.